druntime/src/core/runtime.d

```diff
@@ -105,7 +105,7 @@ private
 }
 
 
-shared static this()
+pragma(crt_constructor) void runtimeInitializer()
 {
     // NOTE: Some module ctors will run before this handler is set, so it's
```

Comment:
This comment would no longer be true.

Reply:
There evidently is an undocumented order dependency in the constructors, which is causing this PR to fail.